generators/app/prompts.js

```diff
@@ -228,30 +268,32 @@ function askForUnitaryEntityOperations(context, done) {
             when: context.props.currentEntity,
             type: 'checkbox',
             name: 'currentEntityComponents',
-            validate: input => (_.isEmpty(input) ? 'you have to choose at least one component' : true),
+            validate: input => (_.isEmpty(input) ? 'You have to choose at least one component' : true),
             message: 'Which components do you want to generate?',
             choices: getConcernedComponents(previousConfiguration, context.props.currentEntity),
```

Comment:
When i try to use incremental mode on a no_entity generation, the prompt always show me all components choices (Producer & consumer). 
I think, it should only provide missing components like for entity components generation.  

![57-PR-review](https://user-images.githubusercontent.com/44546269/83667141-3e94b300-a5ce-11ea-9721-9e18ddece7af.png)

Reply:
You mean if you're typing the same prefix in another incremental mode?

Reply:
Yes, with prefix Test, I choose consumer for the first loop and in second loop for the same prefix it show consumer and producer instead of producer only